src/hazelweb/gui/TermTag.re

```diff
@@ -0,0 +1,42 @@
+module Vdom = Virtual_dom.Vdom;
+
```

Comment:
not obvious to me this should be its own module instead of living in assistant view common or whatever